lib/metanorma/compile.rb

```diff
@@ -252,8 +256,54 @@ def process_extensions(extensions, file, isodoc, options)
       end
     end
 
+    def install_fonts(options)
+      if options[:no_install_fonts]
+        Util.log("[fontist] Skip font installation because" \
+          " --no_install_fonts argument passed", :debug)
```

Comment:
Can we make `--no-install-fonts` instead of `--no_install_fonts `? Hyphenation is command line convention.